template.tpl

```diff
@@ -1395,6 +1342,91 @@ ___TEMPLATE_PARAMETERS___
         "type": "EQUALS"
       }
     ]
+  },
+  {
+    "type": "TEXT",
+    "name": "stackAdaptConversionEventID",
+    "displayName": "StackAdapt Conversion Event Unique ID",
+    "simpleValueType": true,
+    "enablingConditions": [
+      {
+        "paramName": "tagType",
+        "paramValue": "stackAdaptEvent",
+        "type": "EQUALS"
+      }
+    ],
+    "valueValidators": [
+      {
+        "type": "NON_EMPTY"
+      }
+    ]
+  },
+  {
+    "type": "SIMPLE_TABLE",
```

Comment:
I moved these down to the end of the list because they were showing up above the stackadapt conversion event field. It's curious that this didn't seem to affect any of the other types, but I feel like we generally always want the additional event/user properties to show up after all other fields. At least that's my general impression right now.